spec/yaml/swiftnav/sbp/observation.yaml

```diff
@@ -244,11 +243,9 @@ definitions:
           type: u8
           desc: Satellite is healthy?
       - sid:
-          type: u32
+          type: sbp_signal
           desc: |
-            Signal identifier being tracked - values 0x00 through 0x1F represent
-            GPS PRNs 1 through 32 respectively (PRN-1 notation); other values
-            reserved for future use
+            Signal identifier of the satellite beign tracked.
```

Comment:
I'd suggest "Identifier of the signal being tracked (constellation / satellite / band)".